eslint.config.js

```diff
@@ -23,11 +19,6 @@ export default [
       'jsdoc/require-returns-description': 'off',
     },
   },
-  {
-    ...NetflixCommon,
-    files: ['demo/react/**/*.js'],
-    languageOptions: { globals: { ...globals.browser, React: 'readonly', ReactDOM: 'readonly' } },
```

Comment:
By importing react as an esm module — we can declare module-level variables and ditch this carve-out in `eslint` to relax rules for the react demo.